clang/lib/Parse/ParseDecl.cpp

```diff
@@ -194,6 +194,8 @@ void Parser::ParseGNUAttributes(ParsedAttributes &Attrs,
   SourceLocation StartLoc = Tok.getLocation();
   SourceLocation EndLoc = StartLoc;
 
+  llvm::SaveAndRestore<bool> SART(PP.IsInAttr, true);
```

Comment:
add #SYCLomatic_CUSTOMIZATION to protect it?

Reply:
why here set the value to true,  could you add comment?

Reply:
added.